src/Aspire.Hosting.Keycloak/KeycloakResourceBuilderExtensions.cs

```diff
@@ -14,7 +14,10 @@ public static class KeycloakResourceBuilderExtensions
 {
     private const string AdminEnvVarName = "KEYCLOAK_ADMIN";
     private const string AdminPasswordEnvVarName = "KEYCLOAK_ADMIN_PASSWORD";
+    private const string HealthCheckEnvVarName = "KC_HEALTH_ENABLED";
```

Comment:
Comment with a link to the relevant docs? e.g.

```csharp
// As per https://www.keycloak.org/observability/health
```

Reply:
Done!